src/umessage/umessagebuilder.rs

```diff
@@ -698,15 +695,16 @@ impl UMessageBuilder {
     ///
     /// ```rust
     /// use protobuf::{Enum, Message};
-    /// use up_rust::{UCode, UMessageBuilder, UMessageType, UPriority, UStatus, UUri};
+    /// use up_rust::{UCode, UMessageBuilder, UMessageType, UPriority, UStatus, UUIDBuilder, UUri};
     ///
     /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
     /// let invoked_method = UUri::try_from("my-vehicle/cabin/1/rpc.doors")?;
     /// let reply_to_address = UUri::try_from("my-cloud/dashboard/1/rpc.response")?;
-    /// # let request_msg = UMessageBuilder::request(invoked_method.clone(), reply_to_address.clone(), 1000).build();
-    /// // extracting reqid from received Request message for use in building Response message
-    /// let reqid = request_msg.as_ref().unwrap().clone().attributes.as_ref().unwrap().id.as_ref().unwrap().clone();
-    /// let message = UMessageBuilder::response(reply_to_address, reqid, invoked_method)
+    /// let request_id = UUIDBuilder::build();
+    /// // a service implementation would normally use
+    /// // `UUIDBuilder::response_for_request(&request_message.attributes)` instead
+    /// let message = UMessageBuilder::response(reply_to_address, request_id, invoked_method)
+    ///                    .with_message_id(UUIDBuilder::build())
```

Comment:
any particular reason you want to set the message_id explicitly here?

Reply:
My bad -- got caught up in me reverting changes for those tests which do use the request id.